ssh-agency.el

```diff
@@ -202,9 +202,9 @@ ssh-agency always finds the agent without consulting this file."
 (cl-defun ssh-agency-find-socket-from-ss (&key glob regexp)
   "Use `ss' to find an ssh-agent socket matching GLOB and/or REGEXP."
   (catch 'socket
-    (dolist (sock-line (apply #'process-lines
-                              "ss" "--no-header" "--listening" "--family=unix"
-                              (if glob (list "src" glob))))
+    (dolist (sock-line (cdr (apply #'process-lines
```

Comment:
Thanks, could you also add a comment mentioning that we avoid `--no-header` on purpose please?

Reply:
Sure